datafusion/core/tests/parquet/external_access_plan.rs

```diff
@@ -369,45 +379,43 @@ struct TestData {
     file_size: u64,
 }
 
-static TEST_DATA: OnceLock<TestData> = OnceLock::new();
+static _TEST_DATA: OnceLock<TestData> = OnceLock::new();
 
 /// Return a parquet file with 2 row groups each with 5 rows
-fn get_test_data() -> &'static TestData {
-    TEST_DATA.get_or_init(|| {
```

Comment:
The reason to use the `TEST_DATA` thing was to avoid creating the same file over and over again.

Do we need to remove `TEST_DATA`? 

It seems like the core difference is the use of `tempfile_in` rather than the change from static

Reply:
If we aren't going to use `TEST_DATA` anymore perhaps we can remove it (rather than name it starting with `_` to avoid a compiler error) 🤔

Reply:
> The reason to use the `TEST_DATA` thing was to avoid creating the same file over and over again.
> 
> Do we need to remove `TEST_DATA`?
> 
> It seems like the core difference is the use of `tempfile_in` rather than the change from static

There is no chance for delete TEST_DATA. Because there is no main test function to lead minor test cases.